docs/docs/agents/overview.md

```diff
@@ -1,28 +1,36 @@
+- [**Human-in-the-loop control**](./human-in-the-loop.md): Execution can pause *indefinitely* to await human feedback—unlike websocket-based solutions limited to real-time interaction. This enables asynchronous approval, correction, or intervention at any point in the workflow.
+- [**Streaming support**](./streaming.md): Real-time streaming of agent state, model tokens, tool outputs, or combined streams.
+- [**Deployment tooling**](./deployment.md): Includes infrastructure-free deployment tools. The [**LangGraph Platform**](https://langchain-ai.github.io/langgraph/concepts/langgraph_platform/) supports testing, debugging, and deployment.
+    - **[Studio](https://langchain-ai.github.io/langgraph/concepts/langgraph_studio/)**: A visual IDE for inspecting and debugging workflows.
+    - Supports multiple [**deployment options**](https://langchain-ai.github.io/langgraph/tutorials/deployment/) for production.
 
-## Why LangGraph?
+## High-level building blocks
 
-Building agents with LangGraph has these central benefits:
+LangGraph comes with a set of prebuilt components that implement common agent behaviors and workflows. These abstractions are built on top of the LangGraph framework, offering a faster path to production while remaining flexible for advanced customization.
 
-- [**Memory**](./memory.md): Easy to add short-term memory (remember a single conversation) and long-term memory (remember information across conversations) to your agents.
+Using LangGraph for agent development allows you to focus on your application's logic and behavior, instead of building and maintaining the supporting infrastructure for state, memory, and human feedback.
 
-- [**Human-in-the-loop**](./human-in-the-loop.md): Agent execution can be interrupted and resumed, allowing for tool call review, approval and editing via human input.
+## Package ecosystem
 
-- [**Streaming**](./streaming.md): Support for streaming agent state to the user (or developer) over the course of agent's execution. You can stream agent progress, LLM tokens, tool updates, or all of those combined.
+| Package                                    | Description                                                                 | Installation                            |
+|--------------------------------------------|-----------------------------------------------------------------------------|-----------------------------------------|
+| `langgraph-prebuilt` (part of `langgraph`) | Prebuilt components to [**create agents**](./agents.md)                     | `pip install -U langgraph langchain`    |
+| `langgraph-supervisor`                     | Tools for building [**supervisor**](./multi-agent.md#supervisor) agents     | `pip install -U langgraph-supervisor`   |
+| `langgraph-swarm`                          | Tools for building [**swarm**](./multi-agent.md#swarm) agents               | `pip install -U langgraph-swarm`        |
```

Comment:
"swarm agents" is a bit strange - those are not agents, but multi-agent systems

Reply:
same for supervisor

Reply:
supervisor might be fine cause it is an agent. i would just say multi-agent system?